docs/source/markdown/podman-network-create.1.md

```diff
@@ -45,6 +45,10 @@ must be used with a *subnet* option.
 Create a *Macvlan* based connection rather than a classic bridge.  You must pass an interface name from the host for the
 Macvlan connection.
 
+#### **--mtu**
+
+The MTU (integer, optional).
```

Comment:
Chunked this together form Wikipedia, is this correct or needs adjusting?  I think we need a bit more info than what's here at the moment.

```suggestion
The Maximum Transmission Unit (MTU), the size of the largest protocol data unit that can be communicated in a single network-layer transaction (integer, optional).
```

Reply:
Hopefully the middle road is OK, with the new options